py/redrock/external/desi.py

```diff
@@ -33,7 +36,8 @@ def write_zbest(outfile, zbest):
     zbest.meta['EXTNAME'] = 'ZBEST'
     zbest.write(outfile, overwrite=True)
 
-def read_spectra(spectrafiles, targetids=None):
+
+def read_spectra(spectrafiles, targetids=None, spectrum_class=None):
```

Comment:
Please document new `spectrum_class` option.  Consider changing the default to `SimpleSpectrum` instead of None so that read_spectra can be used from interactive analysis work with minimal hassle about needing to know which spectrum class to use.